### ISSUE TITLE
Add CI settings

### ISSUE THREAD

**Issue by yamasakih** [yamasakih/predictiveness-curve#41]
Ref. https://github.com/natsukium/pure-python-library-template/blob/master/.travis.yml

**Comment by yamasakih** [comment]
Install travis command to Mac OS by the following command:

```shell
brew install ruby
echo 'export PATH="/usr/local/opt/ruby/bin:$PATH"' >> ~/.zshrc
gem update --system
gem install travis
```

travis command is found at `/usr/local/lib/ruby/gems/2.6.0/bin`

Ref. https://qiita.com/abetomo/items/462f5e17ecfce1953324 (Japanese)

**Comment by yamasakih** [comment]
Pull request #46

**Comment by yamasakih** [comment]
natsukium bug fix #46.  

Travis CI raised error because `Pipfile.lock` had been not updated.  
It needed the following command:

```shell
pipenv update
```

**Comment by yamasakih** [comment]
It worked. I will close. Good work ☕️
